### PR TITLE
Appearance pane

### LINKED ISSUE #23: Add dark and slim mode
- [x] Add toggles to enable the dark theme as well as slim variations.
- [x] Rename "Backgrounds" tab to "Appearance"

### PR DESCRIPTION
**Pull request by jackpot51**
Rename Background pane to appearance and add dark mode and slim mode toggles

Fixes #23

### CODE REVIEW

File: debian/patches/pop_appearance.patch [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
++      gtk_theme = "Pop";
++      break;
++  }
++
++  if (gtk_theme) {
++    if (panel->interface_settings) {
++      g_settings_set_string(panel->interface_settings, WP_GTK_THEME_KEY, gtk_theme);
++      g_settings_apply(panel->interface_settings);
++    }
++  }
```
Comment (mmstick) [inline debian/patches/pop_appearance.patch:295]: Should this be

```
if (gtk_theme && panel->interface_settings) { }
```

?

**Review by mmstick — DISMISSED** [review]
The code looks good.

File: debian/patches/pop_appearance.patch (hunk 1 above)
Comment (jackpot51) [reply debian/patches/pop_appearance.patch:295]: Sure, I will do that

**Review by mmstick — DISMISSED** [review]

**Comment by jackpot51** [comment]
standard-theme
dark-slim-theme

**Comment by brs17** [comment]
Do we want to change the description of the "Appearance" page?

Currently it says: `Change your background image to a wallpaper or photo`, which seems to be the text searched by GNOME. Maybe we want to include dark theme related text? Or maybe "theme" related text in general?

**Comment by jackpot51** [comment]
Do you have a suggestion?

**Comment by jackpot51** [comment]
I added `Toggle dark mode and slim mode.`

**Review by brs17 — APPROVED** [review]

**Review by mmstick — APPROVED** [review]

**Pull request merged**